projects/aca-content/src/lib/components/libraries/libraries.component.ts

```diff
@@ -31,15 +31,20 @@ import { Store } from '@ngrx/store';
 import { ContentManagementService } from '../../services/content-management.service';
 import { PageComponent } from '../page.component';
 import { AppExtensionService, AppHookService } from '@alfresco/aca-shared';
-import { DocumentListPresetRef } from '@alfresco/adf-extensions';
+import { ContentActionRef, DocumentListPresetRef } from '@alfresco/adf-extensions';
+import { takeUntil } from 'rxjs/operators';
 
 @Component({
-  templateUrl: './libraries.component.html'
+  templateUrl: './libraries.component.html',
+  styleUrls: ['./libraries.component.scss']
 })
 export class LibrariesComponent extends PageComponent implements OnInit {
   isSmallScreen = false;
-
   columns: DocumentListPresetRef[] = [];
+  searchVisibility = false;
+  isMainActionPresent: boolean;
```

Comment:
you copied `isMainActionPresent` from `CreateMenuComponent` but this feature is not used in this component